### PR TITLE
PDA Interface Redesign

### PR DESCRIPTION
**Pull request by AffectedArc07**
**NOTE:** A cache clear may be required to load the new icons

Lets be honest here, the current PDAs are truly disgusting. Pea green monochrome display in 2558?
Things this PR changes:
- The PDA UI now doesn't look like vomit
- The images for the UI have been recoloured (Some even re-designed)
- The crew manifest is now ordered in the same way it is on the lobby screen
- The captain can no longer toggle the door on the syndicate ship

Screenshots:
PDA with all options (Captain's cartridge):
image

PDA with no cartridge:
image

New crew manifest:
image

Things tested:
- PDA messenger
- Crew manifest
- PDA with a pAI in it
- PDA with an ID in it
- PDA with both

:cl: AffectedArc07
imageadd: New icons for the PDA buttons
add: New interface for the PDA
tweak: The crew manifest on the PDA is now ordered
fix: Captain's cartridge no longer has access to the syndicate shuttle door
/:cl:

Why: The current PDA is absolute garbage
Also I finally learnt how to use git properly, thank you flatty

### CODE REVIEW

**Comment by FlattestGuitar** [comment]
This looks swood as heck.

**Comment by AffectedArc07** [comment]
Properly spaced icons example
image

**Comment by Cactus2388** [comment]
it looks SO MUCH BETTER

**Comment by Ralta** [comment]
All my yes.

**Comment by Citinited** [comment]
The pea green PDA interface is classic SS13 though ;)

**Comment by AffectedArc07** [comment]
@Citinited no

File: code/game/objects/items/devices/PDA/PDA.dm [hunk 1]
```diff
@@ -443,7 +433,7 @@ GLOBAL_LIST_EMPTY(PDAs)
 
 //SYNDICATE FUNCTIONS===================================
 
-			if("Toggle Door")
+			if("Toggle ")
```
Comment (FlattestGuitar) [inline code/game/objects/items/devices/PDA/PDA.dm]: Remove the one space here.

File: code/game/objects/items/devices/PDA/PDA.dm [hunk 2]
```diff
@@ -121,152 +121,143 @@ GLOBAL_LIST_EMPTY(PDAs)
...
+					if(cartridge.access & CART_REAGENT_SCANNER)
+						dat += "<li><img src=pda_notes.png class=pda_icon>   <a href='byond://?src=[REF(src)];choice=Reagent Scan'>[scanmode == 3 ? "Disable" : "Enable"] Reagent Scanner</a></li>"
+					if(cartridge.access & CART_ENGINE)
+						dat += "<li><img src=pda_notes.png class=pda_icon>   <a href='byond://?src=[REF(src)];choice=Halogen Counter'>[scanmode == 4 ? "Disable" : "Enable"] Halogen Counter</a></li>"
+					if(cartridge.access & CART_MEDICAL)
+						dat += "<li><img src=pda_scanner.png class=pda_icon>   <a href='byond://?src=[REF(src)];choice=Medical Scan'>[scanmode == 1 ? "Disable" : "Enable"] Medical Scanner</a></li>"
+					if(cartridge.access & CART_ATMOS)
+						dat += "<li><img src=pda_notes.png class=pda_icon>   <a href='byond://?src=[REF(src)];choice=Gas Scan'>[scanmode == 5 ? "Disable" : "Enable"] Gas Scanner</a></li>"
+					if(cartridge.access & CART_REMOTE_DOOR)
+						dat += "<li><img src=pda_rdoor.png class=pda_icon>   <a href='byond://?src=[REF(src)];choice=Toggle Door'Toggle Remote Door</a></li>"
```
Comment (FlattestGuitar) [inline code/game/objects/items/devices/PDA/PDA.dm]: This 'Toggle Door' <a> tag seems all wrong.

File: code/game/objects/items/devices/PDA/cart.dm [hunk 3]
```diff
@@ -189,7 +189,7 @@
 	name = "\improper Value-PAK cartridge"
 	desc = "Now with 350% more value!" //Give the Captain...EVERYTHING! (Except Mime and Clown)
 	icon_state = "cart-c"
-	access = ~(CART_CLOWN | CART_MIME)
+	access = ~(CART_CLOWN | CART_MIME | CART_REMOTE_DOOR)
```
Comment (FlattestGuitar) [inline code/game/objects/items/devices/PDA/cart.dm]: Why'd you remove that?

**Comment by SmokingKilz** [comment]
As nostalgic as the classic green PDA screen is, it just hurts my eyes. I appreciate the work and I fully support this.

File: code/game/objects/items/devices/PDA/cart.dm (hunk 3 above)
Comment (AffectedArc07) [reply code/game/objects/items/devices/PDA/cart.dm]: Technically it removes, since this is a list of carts **NOT** to use

**Pull request merged**